gems/quilt_rails/README.md

```diff
@@ -1,3 +1,11 @@
+# Alpha functionality - do not use in high-traffic production applications
```

Comment:
Is there anything we can link to if they want more details?

Reply:
Added links to the "what the problem is" issue, and the "future architecture discussion" issue.

Also added a snipped about altering existing apps to skip server-side queries.